hack/components/bump-linux-bridge.sh

```diff
@@ -40,10 +41,16 @@ EOF
 echo 'Push the image to KubeVirt repo'
 (
     if [ ! -z ${PUSH_IMAGES} ]; then
-        docker push ${LINUX_BRIDGE_IMAGE_TAGGED}
+        docker push "${LINUX_BRIDGE_IMAGE_TAGGED}"
     fi
 )
 
+if [[ "$(docker-utils::check_image_exists "${LINUX_BRIDGE_IMAGE}" "${LINUX_BRIDGE_TAG}")" ]]; then
```

Comment:
why do you check this only on linux-bridge?

Reply:
Because it is the only one that built locally. And if this version was not pushed yet, we can't get the digest.

Reply:
I think all components can point to simple commits (that are not tagged and thus not pushed) - so maybe we need  to address this on all components. what do you think?

Reply:
At least in the bump scripts, this is the only one that build the image.